src/lseq.rs

```diff
@@ -163,8 +163,7 @@ impl<T: Clone, A: Actor> LSeq<T, A> {
             dot: self.dot.clone(),
             val,
         };
-        // TODO: refactor to follow the library API (don't apply ops immediately)
```

Comment:
Don't we also need to remove L160 as it'll increment the dot, and we just need the next value without incrementing it??

Reply:
I think we need the next op, to have the dot incremented... but you're right locally not

Reply:
yes, we should use `dot: self.dot.inc()`